sos/report/plugins/networking.py

```diff
@@ -183,14 +183,24 @@ def setup(self):
                 "ethtool -a " + eth,
                 "ethtool -c " + eth,
                 "ethtool -g " + eth,
-                "ethtool -e " + eth,
                 "ethtool -P " + eth,
                 "ethtool -l " + eth,
                 "ethtool --phy-statistics " + eth,
                 "ethtool --show-priv-flags " + eth,
                 "ethtool --show-eee " + eth
             ])
 
+            # skip EEPROM collection for 'bnx2x' NICs as this command
+            # can pause the NIC and is not production safe.
+            bnx_output = {
+                "cmd": "ethtool -i %s" % eth,
```

Comment:
In general, ACK to the codechange (if there is an agreement we should avoid on calling the cmd on 'bnx2x' drivers - I havent followed the discussion in the Issue in detail).